src/client/GameWindow.cpp

```diff
@@ -383,27 +388,31 @@ namespace client
     /*!
      * \brief Load all the textures of the map
      */
-     void GameWindow::loadElementTexture()
+    void GameWindow::loadElementTexture()
     {
         std::string folder_path = RESOURCES_PATH "/img/map/element/";
         std::vector<std::string> png_files;
-        DIR* dir = opendir(folder_path.c_str());
+        DIR *dir = opendir(folder_path.c_str());
 
-        struct dirent* entry;
-        while ((entry = readdir(dir)) != nullptr) {
+        struct dirent *entry;
+        while ((entry = readdir(dir)) != nullptr)
+        {
 
-            if (entry->d_name[0] == '.') continue;
+            if (entry->d_name[0] == '.')
+                continue;
 
             std::string filename = entry->d_name;
-            if (filename.size() >= 4 && filename.substr(filename.size() - 4) == ".png") {
-            png_files.push_back(filename);
+            if (filename.size() >= 4 && filename.substr(filename.size() - 4) == ".png")
+            {
+                png_files.push_back(filename);
             }
         }
 
         closedir(dir);
 
         // Affiche les noms de fichiers trouvés
-        for (const std::string& filename : png_files) {
+        for (const std::string &filename : png_files)
+        {
             std::string path = RESOURCES_PATH "/img/map/element/" + filename;
```

Comment:
```suggestion
            std::string path = RESOURCES_PATH ELEMENT_PATH + filename;
```

Reply:
We could add a `#define` here eventually?

Reply:
Done